### PR TITLE
Updating Third Party Bots information

### PR DESCRIPTION
**Pull request by jgonzalezal**

### CODE REVIEW

File: pages/documents/Bots & Automation/Third Party Bot Connectors.md [hunk 1]
```diff
@@ -40,11 +40,12 @@ There are four steps to onboarding a bot.
...
-   2. [Dialogflow V1](https://developers.liveperson.com/bot-connectors-google-dialogflow.html)
-   3. [Dialogflow V2](https://developers.liveperson.com/bot-connectors-google-dialogflow-version-2.html)
-   4. [Amazon Lex](https://developers.liveperson.com/bot-connectors-amazon-lex.html)
-   5. [Microsoft Bot Framework](https://developers.liveperson.com/bot-connectors-microsoft-bot-framework.html)
-   6. [Custom third-party Bots](https://developers.liveperson.com/bot-connectors-custom-third-party-bots.html)
+   2. [Watson Assistant V2](https://developers.liveperson.com/third-party-bots-ibm-watson-assistant-version-2.html)
+   3. [Dialogflow ES](https://developers.liveperson.com/third-party-bots-google-dialogflow-es-introduction.html)
+   4. [Dialogflow CX](https://developers.liveperson.com/third-party-bots-google-dialogflow-cx.html)
+   5. [Amazon Lex](https://developers.liveperson.com/bot-connectors-amazon-lex.html)
+   6. [Microsoft Bot Framework](https://developers.liveperson.com/bot-connectors-microsoft-bot-framework.html)
```
Comment (ahuppert) [inline pages/documents/Bots & Automation/Third Party Bot Connectors.md]: Name and Link to the vendor are outdated (link still works but is not the most recent one)

```suggestion
   6. [Microsoft Direct Line](https://developers.liveperson.com/third-party-bots-microsoft-direct-line-introduction.html)
```

File: pages/documents/Bots & Automation/Third Party Bot Connectors.md [hunk 2]
```diff
@@ -40,11 +40,12 @@ There are four steps to onboarding a bot.
...
+   7. [Custom third-party Bots](https://developers.liveperson.com/bot-connectors-custom-third-party-bots.html)
```
Comment (ahuppert) [inline pages/documents/Bots & Automation/Third Party Bot Connectors.md]: Same here
```suggestion
   7. [Custom Integration](https://developers.liveperson.com/third-party-bots-custom-integration.html)
```

File: pages/documents/Bots & Automation/Third Party Bot Connectors.md [hunk 3]
```diff
@@ -40,11 +40,12 @@ There are four steps to onboarding a bot.
 3. Error handling - define fallback messages and skills to which the conversation will be routed if the bot becomes unavailable during the conversation.
 4. Connect to the bot vendor:
    1. [Watson Assistant](https://developers.liveperson.com/bot-connectors-ibm-watson-assistant.html)
```
Comment (ahuppert) [inline pages/documents/Bots & Automation/Third Party Bot Connectors.md]:
```suggestion
   1. [IBM Watson Assistant](https://developers.liveperson.com/bot-connectors-ibm-watson-assistant.html)
```

File: pages/documents/Bots & Automation/Third Party Bot Connectors.md [hunk 4]
```diff
@@ -40,11 +40,12 @@ There are four steps to onboarding a bot.
...
-   2. [Dialogflow V1](https://developers.liveperson.com/bot-connectors-google-dialogflow.html)
-   3. [Dialogflow V2](https://developers.liveperson.com/bot-connectors-google-dialogflow-version-2.html)
-   4. [Amazon Lex](https://developers.liveperson.com/bot-connectors-amazon-lex.html)
-   5. [Microsoft Bot Framework](https://developers.liveperson.com/bot-connectors-microsoft-bot-framework.html)
-   6. [Custom third-party Bots](https://developers.liveperson.com/bot-connectors-custom-third-party-bots.html)
+   2. [Watson Assistant V2](https://developers.liveperson.com/third-party-bots-ibm-watson-assistant-version-2.html)
```
Comment (ahuppert) [inline pages/documents/Bots & Automation/Third Party Bot Connectors.md]:
```suggestion
   2. [IBM Watson Assistant Version 2](https://developers.liveperson.com/third-party-bots-ibm-watson-assistant-version-2.html)
```

File: pages/documents/Bots & Automation/Third Party Bot Connectors.md [hunk 5]
```diff
@@ -40,11 +40,12 @@ There are four steps to onboarding a bot.
...
+   3. [Dialogflow ES](https://developers.liveperson.com/third-party-bots-google-dialogflow-es-introduction.html)
```
Comment (ahuppert) [inline pages/documents/Bots & Automation/Third Party Bot Connectors.md]:
```suggestion
   3. [Google Dialogflow ES](https://developers.liveperson.com/third-party-bots-google-dialogflow-es-introduction.html)
```

File: pages/documents/Bots & Automation/Third Party Bot Connectors.md [hunk 6]
```diff
@@ -40,11 +40,12 @@ There are four steps to onboarding a bot.
...
+   4. [Dialogflow CX](https://developers.liveperson.com/third-party-bots-google-dialogflow-cx.html)
```
Comment (ahuppert) [inline pages/documents/Bots & Automation/Third Party Bot Connectors.md]:
```suggestion
   4. [Google Dialogflow CX](https://developers.liveperson.com/third-party-bots-google-dialogflow-cx.html)
```

**Review by ahuppert — CHANGES_REQUESTED** [review]
Could you align the existing vendors a bit better with the current developers documentation? (see my comments)
Also do you have in mind the confluence page mentioned in the jira task?

File: pages/documents/Bots & Automation/Third Party Bot Connectors.md (hunk 6 above)
Comment (jgonzalezal) [reply pages/documents/Bots & Automation/Third Party Bot Connectors.md]: Done

File: pages/documents/Bots & Automation/Third Party Bot Connectors.md (hunk 5 above)
Comment (jgonzalezal) [reply pages/documents/Bots & Automation/Third Party Bot Connectors.md]: Done

File: pages/documents/Bots & Automation/Third Party Bot Connectors.md (hunk 4 above)
Comment (jgonzalezal) [reply pages/documents/Bots & Automation/Third Party Bot Connectors.md]: Done

File: pages/documents/Bots & Automation/Third Party Bot Connectors.md (hunk 3 above)
Comment (jgonzalezal) [reply pages/documents/Bots & Automation/Third Party Bot Connectors.md]: Done

File: pages/documents/Bots & Automation/Third Party Bot Connectors.md (hunk 2 above)
Comment (jgonzalezal) [reply pages/documents/Bots & Automation/Third Party Bot Connectors.md]: Done

File: pages/documents/Bots & Automation/Third Party Bot Connectors.md (hunk 1 above)
Comment (jgonzalezal) [reply pages/documents/Bots & Automation/Third Party Bot Connectors.md]: Done

**Comment by jgonzalezal** [comment]
@ahuppert suggentions addressed

**Review by ahuppert — APPROVED** [review]

**Review by fahimmahmoodmir — APPROVED** [review]

**Pull request merged**